dropshot/tests/test_demo.rs

```diff
@@ -201,10 +200,7 @@ async fn test_demo2json() {
     let testctx = common::test_setup("demo2json", api);
 
     /* Test case: optional field */
-    let input = DemoJsonBody {
-        test1: "bar".to_string(),
-        test2: None,
-    };
+    let input = DemoJsonBody { test1: "bar".to_string(), test2: None };
```

Comment:
I wonder if this file got reformatted.  It seems like a lot of this diff is spurious.

Reply:
Looks like it, gone in latest patch